common/src/main/java/zm/gov/moh/common/submodule/form/widget/BasicDrugWidget.java

```diff
@@ -1,18 +1,51 @@
 package zm.gov.moh.common.submodule.form.widget;
 
 import android.content.Context;
-import android.os.Bundle;
+import android.widget.CompoundButton;
+import android.widget.RadioGroup;
+import android.widget.TableLayout;
+import android.widget.TableRow;
 
-import androidx.appcompat.widget.AppCompatTextView;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.AppCompatSpinner;
 
-public class BasicDrugWidget extends BaseWidget {
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import zm.gov.moh.common.R;
+import zm.gov.moh.common.submodule.form.model.Form;
+import zm.gov.moh.common.submodule.form.model.Logic;
+import zm.gov.moh.core.model.DrugObsValue;
```

Comment:
@kaykabwe this import is not resolving, failing at compile time.

Reply:
Added the missing class. Thanks!

Reply:
Noted you are welcome